sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat/chat_completion_with_model_as_platform.ipynb

```diff
@@ -0,0 +1,565 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FineTuning LLM with Model-As-Service\n",
```

Comment:
Wrong title ? Is this about MaaS or MaaP ?

Reply:
Fixed.